src/platform.rs

```diff
@@ -38,6 +38,10 @@ impl PlatformInterface for Platform {
     exit_status.signal()
   }
 
+  fn exit_code_from_signal(signal: i32) -> i32 {
```

Comment:
I think the link to tldp.org only applies on unix, so let's move it here:

```suggestion
  /// See https://tldp.org/LDP/abs/html/exitcodes.html
  fn exit_code_from_signal(signal: i32) -> i32 {
```